src/Microsoft.ServiceFabric.Actors.KVSToRCMigration/MigrationWorker.cs

```diff
@@ -198,7 +204,7 @@ private async Task<long> FetchAndSaveAsync(long startSN, long snCount, long endS
                                 ActorTrace.Source.WriteInfoWithId(
                                     TraceType,
                                     this.TraceId,
-                                    $"Total Keys migrated - StartSN: {startSN}, SNCount: {snCount}, KeysFetched: {kvsData.Count}, KeysMigrated: {keysMigrated}");
+                                    $"Total Keys migrated - StartSN: {startSN}, SNCount: {snCount}, KeysFetched: {enumerationResponse.KeyValuePairs.Count}, KeysMigrated: {keysMigrated}");
```

Comment:
Suggestion: We should capture and emit telemetry on time spent in SaveStateAsync and PostHydrationValidationAsync. This will help us present detailed stats on performance of Migration operation.

Reply:
TODO